### PR TITLE
Draft 2-week announcement

### PR DESCRIPTION
**Pull request by thatdocslady**
Includes:

- Schedule (still needs to be updated)
- Workshop reminder
- Social events
- Thanks to sponsors (are we missing anything?)
- Tickets sold out + waiting list

Should we include anything else?

Also, my live staging is totally broken for some reason, I deleted and cloned the repo again because of other trouble and now it doesn't work at all (404).

### CODE REVIEW

File: docs/conf/na/2017/news/schedule-social.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+.. post:: May 1, 2017
+   :tags: 2017, portland, schedule, social
+
+Announcing Full Schedule and Social Events
+==========================================
+
+Our Portland conference is just two weeks away and we hope you're getting excited!
+
+We are officially sold-out but you can sign up for the `waiting list <https://ti.to/writethedocs/write-the-docs-na-2017>`_ in case we have any last-minute cancellations.
```
Comment (ericholscher) [inline docs/conf/na/2017/news/schedule-social.rst]: Should probably drop the waiting list bit, as when this goes out folks will probably be too late, and I don't want to deal with more waiting list people.

File: docs/conf/na/2017/news/schedule-social.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Full Presentation Schedule
+--------------------------
+
+Our website is now updated with the `final schedule <http://www.writethedocs.org/conf/na/2017/schedule/>`_ for the presentations, so you can start planning your time at the conference.
+
+Soon we will also publish the timeslots for the unconference sessions as well as a preliminary list of Writing Day projects that you can hack on.
```
Comment (ericholscher) [inline docs/conf/na/2017/news/schedule-social.rst]: `s/Soon/Next week/`?

File: docs/conf/na/2017/news/schedule-social.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+The response to the new workshops has been great and we're nearly at full occupancy for both workshops. If you're still on the fence, don't wait to secure your spot.
+
+You can still grab the last tickets for the workshops on our `ticket website <https://ti.to/writethedocs/write-the-docs-na-2017>`_.
+
+Social Events
+-------------
+
+This year our main social event will be held again at the `Jackknife <https://goo.gl/maps/hvYkv6RU4qD2>`_ at 614 SW 11th Ave, on **Monday evening at 7pm**. Drinks and light snacks will be provided.
+
+We're also happy to invite you to an after-conference hangout sponsored by our friends at New Relic. The hangout will be at `New Relic <https://www.meetup.com/Write-The-Docs-PDX/events/239146623/>`_ at 6:30pm.Drinks and light snacks will be provided.
```
Comment (ericholscher) [inline docs/conf/na/2017/news/schedule-social.rst]: Needs a space `30pm.Drinks`

File: docs/conf/na/2017/news/schedule-social.rst [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+And lastly, there are still a few spots left for the hike, you can register for it `here <https://ti.to/writethedocs/write-the-docs-na-2017>`_.
```
Comment (ericholscher) [inline docs/conf/na/2017/news/schedule-social.rst]: The hike is now full at 50 people -- we'll probably need to break it up into 2, but still trying to figure out logistics -- but we should probably mention it selling out.

File: docs/conf/na/2017/news/schedule-social.rst (hunk 3 above)
Comment (ericholscher) [reply docs/conf/na/2017/news/schedule-social.rst]: Also should include the address for new relic (111 SW 5th Ave, #2700, 27th floor)

File: docs/conf/na/2017/news/schedule-social.rst [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+Thanks to Our Sponsors
+----------------------
+
+Every year we are fortunate to have the support of our communities and organizations, which help to keep our ticket prices low, the value to attendees high, and enable us to offer scholarship tickets to increase diversity.
+
+We'd like to take this opportunity to thank the `sponsors <http://www.writethedocs.org/conf/na/2017/sponsor/>`_ of this year's Portland conference:
+
+* **`Google <https://google.com>`_**
+* **`Wordpress <https://wordpress.com/>`_**
```
Comment (ericholscher) [inline docs/conf/na/2017/news/schedule-social.rst]: Not sure if this works. One of the stupid bits of RST.

**Comment by thatdocslady** [comment]
@ericholscher implemented your feedback, right now it looks like the schedule PR #208 is blocking this PR because of a question by @plaindocs that I can't help with :(

**Comment by plaindocs** [comment]
So, publishing the schedule without links is meh but not a deal breaker I think. Lemme take a quick look.

**Comment by plaindocs** [comment]
Urgh - I see the issue. Text parsing. I reckon we publish without links if necessary, and Eric or I will fix, sooner rather than later.

**Comment by thatdocslady** [comment]
@plaindocs so now what, can I merge the schedule PR?

**Comment by plaindocs** [comment]
So my comment is that the schedule does not link to the abstracts and they should. Nut now I've looked at the code I know *why* they don't.

And rather than block the announcement, I reckon we publish the schedule, and the announcement and we'll link up the schedule later.

That sound ok to you?

**Pull request merged**